glib-sys/src/lib.rs

```diff
@@ -648,24 +651,8 @@ pub const G_DATE_BAD_YEAR: c_int = 0;
 pub const G_DIR_SEPARATOR: c_int = 92;
 pub const G_DIR_SEPARATOR_S: *const c_char = b"\\\0" as *const u8 as *const c_char;
 pub const G_E: c_double = 2.718282;
-pub const G_GINT16_FORMAT: *const c_char = b"hi\0" as *const u8 as *const c_char;
-pub const G_GINT16_MODIFIER: *const c_char = b"h\0" as *const u8 as *const c_char;
-pub const G_GINT32_FORMAT: *const c_char = b"i\0" as *const u8 as *const c_char;
-pub const G_GINT32_MODIFIER: *const c_char = b"\0" as *const u8 as *const c_char;
-//pub const G_GINT64_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINT64_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
 pub const G_GNUC_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
 pub const G_GNUC_PRETTY_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
```

Comment:
These are also both useless, needs a C compiler to do something

Reply:
done